MuMuPiKPAT/test/sanjay/selector/psiPrimePiK_MC.C

```diff
@@ -692,6 +697,18 @@ void psiPrimePiK_MC::SlaveBegin(TTree * /*tree*/)
       mva_variables_all->Branch(mva_vars_name[i], &mva_vars_all[i], mva_vars_name[i]+"/I" );
     }
   }
+
+  vector< TString > hardcuts_vars_name;
+  hardcuts_vars_name.push_back("KPi_mass"); hardcuts_vars_name.push_back("PsiPi_mass"); hardcuts_vars_name.push_back("cosTheta_psi"); hardcuts_vars_name.push_back("phi");
```

Comment:
Do not hardcode variables names but remove "_gen" from https://github.com/lecriste/Z4430/pull/1/files#diff-2aef6870c472c96f6a5aec4107f14debR677

Reply:
I didn't do that because mva_vars_name has more elements that are not needed for hardcuts_vars_name. It has only the AA variables, there is not a one-to-one correspondence.
Can I keep them?

Reply:
Just use the ones you need:
```
for (Int_t i=mva_vars_name-index-of-first-variable-you-need; i<mva_vars_name-index-of-last-variable-you-need; ++i) {
remove "_gen";
}
```